guides/executeTrade.md

```diff
@@ -1,26 +1,58 @@
-:warning: These code snippets are just examples and the data should never be used as is :warning:
+---
+title: "Executing a trade"
+group: Guides
+---
+
+> [!WARNING]
+>
+> These code snippets are just examples and the data should never be used as is
 
 # How to create a Taker order and execute a trade
 
-Trades are executed on-chain by matching a `Maker` order with a `Taker` order. The maker order can be retrieved from the API, see [get v2 orders](https://looksrare.dev/v2/reference/getorders) for more details. While the taker order can be obtained by calling the `createTaker` method as shown here:
+Trades are executed on-chain by matching a `Maker` order with a `Taker` order. The maker order can be retrieved from the [graphQL API](https://staging-api.hypercerts.org/v1/graphql). The order used here has the same structure as a [maker order](../src/types.ts#L116), but with an added field `signature: string`. While the taker order can be obtained by calling the `createTaker` method as shown here:
```

Comment:
Link to production API, or both. But staging could always be broken and not used as reference

Reply:
Addressed